### DISCUSSION TITLE
ArgoCD notification on Google Chat

### DISCUSSION THREAD

**Discussion by PhamQuang-512** [argoproj/argo-cd#17826] [General]
i have following this documentation about setting template to send notify to google chat https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/services/googlechat/
But I can only send simple messages, not card messages. When checking the logs of the argocd notification server, it shows "time="2024-04-13T04:55:55Z" level=info msg="Trigger test-google-chat result: [{[0].y7b5sbwa2Q329JYH755peeq-fBs 2024-04- 13T04:52:36Z [test-google-chat] true}]" resource=argo-cd/service-platform-uat-user-setting-public" 
Here is my template:
![image](https://github.com/argoproj/argo-cd/assets/77691572/45199d89-fcac-42cf-a39d-92715cb2a681)

**Comment by hvmzx** [comment]
I have the same issue, did you find a solution to this ?  I also could only send the title with the first widget, but the moment I write the second widget from repository to author, it throws me errors.

Apparently we can send JSONs but there is no example in the doc ...

**Comment by S1mpleOW** [comment]
i have the same issue; I cannot send any message via Google Chat.
